src/test/regress/expected/failure_connection_establishment.out

```diff
@@ -203,7 +204,7 @@ SELECT citus.mitmproxy('conn.delay(500)');
 (1 row)
 
 INSERT INTO products VALUES (100, '100', 100);
-ERROR:  could not establish any connections to the node localhost:xxxxx after 400 ms
+ERROR:  connection to the remote node localhost:xxxxx failed
```

Comment:
can you try running this test with shard_replication_factor = 1? With replication factor = 2, we are killing the "remote lock" connection, not the actual execution lock. And, the error message becomes much less useful.

For our primary use-case, replication factor = 1, it'd be great to show we can give the proper error message